cursesman/entities.py

```diff
@@ -214,21 +217,27 @@ def burnFuse(self):
             self.explode()
 
     def explode(self):
-        self.exploded = True
-        self.explosions = [
-            Explosion(self.x+dx, self.y+dy, col=self.col)
-            for dx in range(-self.power*4, self.power*4+4, 4)
-            for dy in range(-self.power*4, self.power*4+4, 4)
-            if 0 in [dx, dy]
-        ]
-        threading.Timer(0.1, self.die).start()
+        #this way the list is ordered in directional groups, from the inside out.
+        logging.warning("power = " + str(self.power))
+        explosions = []
+        
+        #left explosions
+        for p in range (1, self.power+1):
+            explosions.append(Explosion(self.x-4*p, self.y, col=self.col))
+            explosions.append(Explosion(self.x+4*p, self.y, col=self.col))
+            explosions.append(Explosion(self.x, self.y-4*p, col=self.col))
+            explosions.append(Explosion(self.x, self.y+4*p, col=self.col))
 
```

Comment:
we dont make an explosion on the bimbino, you should add an extra line before the loop or something